src/node.cc

```diff
@@ -4188,6 +4188,10 @@ void Init(int* argc,
   V8::SetFlagsFromString(NODE_V8_OPTIONS, sizeof(NODE_V8_OPTIONS) - 1);
 #endif
 
+  // Allow for environment set preserving symlinks
+  if (secure_getenv("NODE_PRESERVE_SYMLINKS")) {
+     config_preserve_symlinks = true;
+  }
```

Comment:
I'd write it as e.g.:
```c++
if (auto var = secure_getenv("NODE_PRESERVE_SYMLINKS"))
  config_preserve_symlinks = (*var != '\0');
```
An empty string is false, everything else is true.  That includes blank strings but I think that is an acceptable functionality vs. complexity trade-off.